dbctx/context.go

```diff
@@ -2,32 +2,38 @@ package dbctx
 
 import (
 	"fmt"
-	"github.com/jinzhu/gorm"
-	_ "github.com/jinzhu/gorm/dialects/postgres"
 	"regexp"
+
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/postgres" // Import postgres dialect for GORM.
 )
 
 var (
 	databaseURL string
 )
 
 type (
```

Comment:
Didn't even know that you can put multiple structs in a single type statement. Cool!